native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java

```diff
@@ -0,0 +1,204 @@
+                        moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                    } else {
+                        if (node.identifier().toString().trim().equals(KEY_PORT)) {
+                            existingNodes.add(node.identifier().toString().trim());
+                            moduleMembers = moduleMembers.add(SampleNodeGenerator.createNumericKV(
+                                    node.identifier().toString().trim(),
+                                    defaultValues[indexOf(nodeMap,
+                                            node.identifier().toString().trim())], null));
+                        } else {
+                            existingNodes.add(node.identifier().toString().trim());
+                            moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV(
+                                    node.identifier().toString().trim(),
+                                    defaultValues[indexOf(nodeMap,
+                                            node.identifier().toString().trim())], null));
+                        }
+                    }
+                } else {
+                    moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                }
+            } else if (member instanceof TableNode || member instanceof TableArrayNode) {
+                if (!existingNodes.isEmpty() || !moduleMembers.isEmpty()) {
+                    moduleMembers = addNewLine(moduleMembers, 2);
+                    moduleMembers = poulateRemaining(moduleMembers, existingNodes);
+                    moduleMembers = addNewLine(moduleMembers, 1);
+                    isTableEntry = true;
+                    moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                } else {
+                    isTableEntry = true;
+                    moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                }
+            }
+        }
+        if (existingNodes.isEmpty() && isTableEntry == true) {
+            moduleMembers = addNewLine(moduleMembers, 2);
+            moduleMembers = moduleMembers.add(SampleNodeGenerator.createTable(name, null));
+            moduleMembers = poulateRemaining(moduleMembers, existingNodes);
+            moduleMembers = addNewLine(moduleMembers, 1);
+            
+        } else if (existingNodes.size() != 5) {
+            moduleMembers = addNewLine(moduleMembers, 2);
+            moduleMembers = poulateRemaining(moduleMembers, existingNodes);
+        }
+
+        Token eofToken = AbstractNodeFactory.createIdentifierToken("");
+        DocumentNode documentNode = NodeFactory.createDocumentNode(moduleMembers, eofToken);
+        TextDocument textDocument = TextDocuments.from(documentNode.toSourceCode());
+        SyntaxTree syntaxTreeFinal = SyntaxTree.from(textDocument);
+        return syntaxTreeFinal;
```

Comment:
```suggestion
        return SyntaxTree.from(textDocument);
```

Reply:
Updated in [e4c809d](https://github.com/ballerina-platform/persist-tools/pull/9/commits/e4c809da564408437e379bbe491d495ca2cbd2e8)